home-manager/po/lt.po

```diff
@@ -0,0 +1,186 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR Home Manager contributors
+# This file is distributed under the same license as the Home Manager package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Home Manager\n"
+"Report-Msgid-Bugs-To: https://github.com/nix-community/home-manager/issues\n"
+"POT-Creation-Date: 2022-03-26 15:08+0100\n"
+"PO-Revision-Date: 2023-01-09 11:33+0000\n"
+"Last-Translator: Kornelijus Tvarijanavičius <kornelijus@tvaria.com>\n"
+"Language-Team: Lithuanian <https://hosted.weblate.org/projects/home-manager/"
+"cli/lt/>\n"
+"Language: lt\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n % 10 == 1 && (n % 100 < 11 || n % 100 > "
+"19)) ? 0 : ((n % 10 >= 2 && n % 10 <= 9 && (n % 100 < 11 || n % 100 > 19)) ? "
+"1 : 2);\n"
```

Comment:
Woah

Reply:
Lol wtf https://en.wikipedia.org/wiki/Lithuanian_grammar#Noun_modification_by_numeral

Reply:
Pretty wild. Polish has some similar rules. Dunno if I would manage to learn that 😃